src/mem/alloc.h

```diff
@@ -47,7 +47,7 @@ namespace snmalloc
   class FastFreeLists
   {
   protected:
-    FreeListHead small_fast_free_lists[NUM_SMALL_CLASSES];
+    FreeListIter small_fast_free_lists[NUM_SMALL_CLASSES];
```

Comment:
Nice way to emphasize that these are populated all at once and consumed piecemeal.